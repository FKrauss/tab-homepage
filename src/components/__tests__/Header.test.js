/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Header', () => {
  it('renders without error', () => {
    const Header = require('../Header').default
    shallow(<Header />)
  })

  it('matches expected snapshot', function() {
    const Header = require('../Header').default
    const wrapper = shallow(<Header />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
