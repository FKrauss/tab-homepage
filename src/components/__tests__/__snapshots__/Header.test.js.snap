// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches expected snapshot 1`] = `
<div
  style={
    Object {
      "paddingBottom": 20,
      "paddingLeft": 40,
      "paddingRight": 40,
      "paddingTop": 20,
    }
  }
>
  <GatsbyLink
    to="/"
  >
    <img
      src="test-file-stub"
      style={
        Object {
          "height": 40,
        }
      }
    />
  </GatsbyLink>
</div>
`;
